src/components/content/order/formElements/NumberInput.tsx

```diff
@@ -3,25 +3,45 @@
  * SPDX-FileCopyrightText: Huawei Inc.
  */
 
-import { NumberInputEventHandler, DeployParam } from './CommonTypes';
+import { NumberInputEventHandler, DeployParam, DeployJsonSchema } from './CommonTypes';
 import { Form, InputNumber } from 'antd';
+import React from 'react';
+import { Rule } from 'rc-field-form/lib/interface';
 
 export function NumberInput({
     item,
     onChangeHandler,
 }: {
     item: DeployParam;
     onChangeHandler: NumberInputEventHandler;
-}): JSX.Element {
+}): React.JSX.Element {
+    const ruleItems: Rule[] = [{ required: item.mandatory }, { type: 'number' }, { min: 0 }];
+    let minimum: number;
+    let maximum: number;
+
+    const setValidator = (_: unknown, value: number) => {
+        if (value <= minimum) {
+            return Promise.reject(new Error(`Value does not satisfy criteria minimum ${minimum}!`));

```

Comment:
currently the error is shown multiple times.. also has some "undefined" error.

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/e7a4aeea-3906-4c9e-9763-00877c7b16c4)